doc/user/README.md

```diff
@@ -3,29 +3,7 @@
 This is the root directory of the Materialize user documentation, which are
 rendered by [Hugo] locally and published by CI to <https://materialize.com/docs>.
```

Comment:
"which is", not "which are" (suspect someone changed "docs" to "documentation" at some point.